src/vue/modules/loyalty-points-statistics/components/viewers/incoming-volume-viewer.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="incoming-volume-viewer">
+    <line-chart
+      v-if="isLoaded"
+      id="incoming-volume-chart"
+      :data="incomingVolumes"
+      :currency="point"
+    />
+
+    <load-spinner
+      v-else-if="!isLoadFailed"
+      message-id="loyalty-points.statistics.loading-msg"
```

Comment:
Move these translations to the `incoming-volume-viewer` scope

Reply:
Moved